### PR TITLE
opts/weightdevice: fix typo

### PR DESCRIPTION
**Pull request by Mashimiao**
Signed-off-by: Ma Shimiao mashimiao.fnst@cn.fujitsu.com

### CODE REVIEW

File: opts/weightdevice.go [hunk 1]
```diff
@@ -35,7 +35,7 @@ func (opt *WeightdeviceOpt) Set(val string) error {
 	return nil
 }
 
-// String returns Ulimit values as a string.
+// String returns Weightdevice values as a string.
```
Comment (calavera) [inline opts/weightdevice.go]: `WeightdeviceOpt`?

Comment (Mashimiao) [inline opts/weightdevice.go]: fine

**Comment by vdemeester** [comment]
LGTM 🐶

**Comment by runcom** [comment]
LGTM

**Pull request merged**